hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -814,6 +814,41 @@ message KeyLocationList {
     optional bool isMultipartKey = 4 [default = false];
 }
 
+/**
+ * Checksum algorithms/types used in Ozone
+ * Make sure this enum's integer values match enum values' id properties defined
+ * in org.apache.hadoop.util.DataChecksum.Type
+ */
+enum ChecksumTypeProto {
+  CHECKSUM_NULL = 0;
+  CHECKSUM_CRC32 = 1;
+  CHECKSUM_CRC32C = 2;
+}
+
+enum FileChecksumTypeProto {
+  COMPOSITE_CRC = 1;  // Chunk-independent CRC, optionally striped
+  MD5CRC = 2;  // BlockChecksum obtained by taking the MD5 digest of chunk CRCs
```

Comment:
This order is different than HDFS.  Shall we make it consistent?

Reply:
@rakeshadr , is this COMPOSITE_CRC for EC strips only?

Reply:
Composite represents - Calculate a single CRC that represents the entire file and is independent of the lower-level granularity of chunk CRCs. Its not only for EC, instead files of different layout (different block size, EC).

Reply:
I see.  Thanks @rakeshadr for the explanation.